features/fake_time.feature

```diff
@@ -0,0 +1,128 @@
+Feature: Allow time to be faked by utilities such as sinon.useFakeTimers
+  Background: Before and After hooks to enable faking time.
+    Given a file named "features/support/hooks.js" with:
+    """
+    var sinon = require('sinon');
+    var hooks = function () {
+
+      this.Before({tags: ['@fakedTime']}, function(scenario, callback) {
+        this.clock = sinon.useFakeTimers();
+        callback();
+      });
+
+      this.After({tags: ['@fakedTime']}, function(scenario, callback) {
+        this.clock.restore();
+        callback();
+      });
```

Comment:
you can just omit the callback in each of these since they are synchronous. Thus you can omit scenario as well